tests/unittests/Makefile

```diff
@@ -226,12 +226,18 @@ BASELIBS += $(UNIT_TESTS:%=$(BINDIR)/%.a)
 
 INCLUDES += -I$(RIOTBASE)/tests/unittests/common
 
+# list of boards to run CI tests on
+TEST_WHITELIST += all
+
 include $(RIOTBASE)/Makefile.include
 
 .PHONY: $(UNIT_TESTS)
 
 all:
 
+info-unittests:
+	@echo $(UNIT_TESTS)
+
```

Comment:
It won't be useful to give info for any test? Why only for unittests (at least in the examples you're giving).

Reply:
Only unittests are split into tests-*. This info target outputs that information. I want to use that to split up the unittests into seperate binaries for platforms where all tests don't fit.